win32ss/user/ntuser/nonclient.c

```diff
@@ -547,7 +551,7 @@ DefWndDoSizeMove(PWND pwnd, WORD wParam)
               //
               unmodRect = newRect;
 
-	      /* determine the hit location */
+          /* determine the hit location */
```

Comment:
```suggestion
              /* determine the hit location */
```

Reply:
Actually, the following 60 lines are incorrectly indented while this line is technically correct.